example/FileUploadDragDrop/index.html

```diff
@@ -84,7 +84,7 @@ <h3 class="page-header">Lotus Web Components DragDropFileCollectionView Control<
         is attached to each tag instance. Open your console to observe the file upload callbacks. In order to upload files your callbacks should trigger upload behaviors
         and update the files load status. See the commnets in the code example at the bottom of this page for psuedo code.
     </p>
-    <x-lotus-upload data-template-url="templates/fileUpload.html"
+    <x-lotus-upload data-template-url="templates/skinned-fileUploadDragDrop.html"
```

Comment:
can you move your changes to fileUpload.html and remove skinned-fileUploadDragDrop.html. We don't need the old files. Same for all the skins.